components/VoteParticipation/VoteParticipation.tsx

```diff
@@ -81,6 +87,28 @@ const Header = styled.div`
   font: var(--text-xxl);
   width: 921px;
   letter-spacing: -0.01em;
+  position: relative;
+  background: -webkit-radial-gradient(center, 50% 50%, red, black);
```

Comment:
I see this is wip so ignore if you're already gonna do this.

seems like you've used the standard css `red` and `black` here, make sure to change this to the brand color vars.

Reply:
yeah this is WIP, going to move on and tackle this later.